oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -512,6 +512,17 @@ public APIGatewayProxyResponseEvent handleRequest(
                                 > 0) {
                     userSession.setCurrentCredentialStrength(lowestRequestedCredentialTrustLevel);
                 }
+                var currentCredentialStrength =
+                        userInfo.getStringClaim(
```

Comment:
It looks like getClaim and getStringClaim do basically the same thing but different. However, we haven't used getStringClaim anywhere. Is there a reason here to use this one? Otherwise I think better to keep with getClaim so we can easily find all uses of it

Reply:
Think they basically do the same - just a bit more tidy. We do use it elsewhere in the repo. Not consistent in the handler but think we should push to switch it over to `getClaimString` as we do use it elsewhere.

Reply:
When I search the codebase, I can't see any uses of getClaimString. Where are you seeing it be used? 

I do agree though. There are instances of using `getClaim(claimName, String.class)` which is the same, but `getClaimString` is cleaner of course. Might be an easy strong-typing win; a 1 pointer for the backlog perhaps.

Reply:
It used for the JWTclaims set, claimsSet in the AuthoriseAccessTokenHandler and in a few tests. Its not too much. I guess should we leave this as `getStringClaim` as a reminder and do the rest in another ticket . Or keep it consistent for now and switch it back to `getClaim`.